packages/preface/preface.1.0.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+
+maintainer: "xaviervdw@gmail.com"
+authors: [
+  "Didier Plaindoux <d.plaindoux@free.fr>"
+  "Pierre Ruyter <grimfw@gmail.com>"
+  "Xavier Van de Woestyne <xaviervdw@gmail.com>"
+]
+
+license: "MIT"
+tags: ["library" "standard" "monad"]
+homepage: "https://github.com/xvw/preface"
+dev-repo: "git+https://github.com/xvw/preface.git"
+bug-reports: "https://github.com/xvw/preface/issues"
+
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name ] {with-test}
```

Comment:
```suggestion
  [ "dune" "runtest" "-p" name ] {with-test & !base-effects:installed}
```

Reply:
This disables the tests for the `effects` compiler (currently this is 4.12-domains-effects I think).

Reply:
Thank!